composer.json

```diff
@@ -35,14 +35,14 @@
     },
     "require-dev": {
         "doctrine/coding-standard": "^10.0",
-        "gajus/dindent": "^2.0.2",
         "phpstan/phpstan": "^1.9",
         "phpstan/phpstan-deprecation-rules": "^1.0",
         "phpstan/phpstan-phpunit": "^1.2",
         "phpstan/phpstan-strict-rules": "^1.4",
         "phpunit/phpunit": "^9.5",
         "symfony/css-selector": "4.4 || ^5.2 || ^6.0",
-        "symfony/dom-crawler": "4.4 || ^5.2 || ^6.0"
+        "symfony/dom-crawler": "4.4 || ^5.2 || ^6.0",
+        "ext-dom": "*"
```

Comment:
This should be the first item of this list.

Reply:
do we have a tool to normalize the composer.json? In TYPO3 we use `ergebnis/composer-normalize` but it is not installed here